plat/drivers/ofw/fdt.c

```diff
@@ -315,18 +315,22 @@ int fdt_node_offset_by_compatible_list(const void *fdt, int startoffset,
 int fdt_node_offset_idx_by_compatible_list(const void *fdt, int startoffset,
 			  const char * const compatibles[], int *index)
 {
-	int idx, offset;
+	int idx, min_offset = INT_MAX, compatible_idx;
 
 	for (idx = 0; compatibles[idx] != NULL; idx++) {
-		offset = fdt_node_offset_by_compatible(fdt, startoffset,
+		int offset = fdt_node_offset_by_compatible(fdt, startoffset,
```

Comment:
Maybe we could declare this at the beginning of the function, like the other local variables?

Reply:
Good point, thanks!